docs/application/native/guides/internationalization/i18n.md

```diff
@@ -807,10 +807,10 @@ PluralFormat deals with this by diving the problem into two parts:
 ### Usage of Plural Format
 
 The following lists the usage of PluralFormat:
-1. You will need to use Plural module to properly use PluralFormat module.
-2. To get default set of rules, you can use constructor, which takes local object as parameter.
-3. A message pattern is specified by passing argument to constructor, or set it explicitly by using `18n_plural_format_apply_pattern()`.
-4. To obtain formatted message, you need to use one of `i18n_plural_format_format_*`, which also takes
+- You will need to use Plural module to properly use PluralFormat module.
+- To get default set of rules, you can use constructor, which takes local object as parameter.
+- A message pattern is specified by passing argument to constructor, or set it explicitly by using `i18n_plural_format_apply_pattern()`.
+- To obtain formatted message, you need to use one of `i18n_plural_format_format_*`, which also takes
 a number object and returns corresponding message based on the defined rules.
```

Comment:
There are some more comments that needs to be fixed.

Line 1629: Please change “Managing Iteration Using Ubrk” to “Manage Iteration Using Ubrk”
Line 1649: Please fix the number formatting
Line 2110 to 2128: Please can you check for the spacing from the specified line number
Why is the word function orphan?
Line 2439 to 2446: Please can you remove additional whitespace between each word.
Line 2801 to 2809: Please can you fix broken code
Line 3016:
To retrieve conversion constants for a specific time scale, use the `i18n_utmscale_get_time_scale_value()` function with the values of the `i18n_utmscale_value_e` enumeration (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__UTMSCALE__MODULE.html#ga34893d20b446359e32767b57e6cdde29) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__UTMSCALE__MODULE.html#ga34893d20b446359e32767b57e6cdde29) applications):

`int64_t value = NULL;i18n_utmscale_get_time_scale_value(I18N_UTMSCALE_JAVA_TIME, I18N_UTMSCALE_VALUE_UNITS, &value);/* Returns 10000, the number of universal time units (ticks) in each Java time unit (milliseconds) */`

Please can you add after the code and text:
To retrieve conversion constants for a specific time scale, use the `i18n_utmscale_get_time_scale_value()` function with the values of the `i18n_utmscale_value_e` enumeration (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__UTMSCALE__MODULE.html#ga34893d20b446359e32767b57e6cdde29) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__UTMSCALE__MODULE.html#ga34893d20b446359e32767b57e6cdde29) applications):

`int64_t value = NULL;i18n_utmscale_get_time_scale_value(I18N_UTMSCALE_JAVA_TIME, I18N_UTMSCALE_VALUE_UNITS, &value);/* Returns 10000, the number of universal time units (ticks) in each Java time unit (milliseconds) */`

Line 3018: 
To convert the retrieved `int64_t` Java time to universal time scale:`int64_t universal_time = NULL;i18n_utmscale_from_int64(value, I18N_UTMSCALE_ICU4C_TIME, &universal_time);`

Please can you add space between text and code:
To convert the retrieved `int64_t` Java time to universal time scale:

`int64_t universal_time = NULL;i18n_utmscale_from_int64(value, I18N_UTMSCALE_ICU4C_TIME, &universal_time);`

Line 3029 to 3037: Please can you remove additional whitespace between each word.
Line 3131: 
To convert the version number from hexadecimal format into a string in dotted decimal format, use the `i18n_uversion_to_string()` function:`char *decimal_version;i18n_uversion_to_string(array, decimal_version);`
Please can you add space between text and code:
To convert the version number from hexadecimal format into a string in dotted decimal format, use the `i18n_uversion_to_string()` function:
`char *decimal_version;i18n_uversion_to_string(array, decimal_version);`

Line 3132:
To convert the version number from a string in dotted decimal format to hexadecimal format, use the `i18n_uversion_from_string()` function:`char *decimal_version = "57.1";i18n_uversion_from_string(decimal_version, version);`If your source string is of the `i18n_uchar` type, use the `i18n_uversion_from_ustring()` function instead.
Please can you break this sentence as:
To convert the version number from a string in dotted decimal format to hexadecimal format, use the `i18n_uversion_from_string()` function:
`char *decimal_version = "57.1";i18n_uversion_from_string(decimal_version, version);
`If your source string is of the `i18n_uchar` type, use the `i18n_uversion_from_ustring()` function instead.

Line 289: Please can you change the heading to “ Locale-Sensitive Operations with Ulocale”
Line 790: Please can you remove colon from the heading
Line 817: Please can you remove colon from the heading
Line 870: Please can you remove colon from the heading
From Line 883 onwards: Please can you change all the heading from the specified line number to non-gerund format.
That is usage of “ing” must be avoided. 
For example: Please change “Managing Characters and Strings” to “Manage Characters and Strings”
Line 909: Please can you change “2” to “two”x`

Line 1087: Please can you change “2” to “two”
Line 1432: Please can change you provide space where ever it is necessary as shown in the following:
To set a date in the Ucalendar, use the `i18n_ucalendar_set_date_time()` function. In the following example, the date is set as 1 July 2014, 9:00:00. To define the month, you can use numbers (month reduced by 1, such as 0 for January and 1 for February), or to avoid mistakes, the values of the `i18n_ucalendar_months_e` enumeration (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__UCALENDAR__MODULE.html#ga094cacb2ef9ee4805e42e276fec5ae2f) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__UCALENDAR__MODULE.html#ga094cacb2ef9ee4805e42e276fec5ae2f) applications).`i18n_ucalendar_set_date_time(ucalendar, 2014, I18N_UCALENDAR_JULY, 1, 9, 0, 0);
`To set a date using milliseconds from the epoch, use the `i18n_ucalendar_set_milliseconds()` function:`i18n_udate udate;/* udate must be set */i18n_ucalendar_set_milliseconds(ucalendar, udate);`To add a specified period to the Ucalendar, use the `i18n_ucalendar_add()` function.Specify the date field to modify (such as year, week, or day) using the `i18n_ucalendar_date_fields_e` enumeration (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__UCALENDAR__MODULE.html#gaee345f9992035a07732d16d69c41c192) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__UCALENDAR__MODULE.html#gaee345f9992035a07732d16d69c41c192) applications) as the second parameter, and the amount to modify as the third parameter (use a negative value to subtract from the existing value).